package.nls.hindi.json

```
@@ -0,0 +1,31 @@
+{
+    "python.command.python.sortImports.title": "आयात क्रमबद्ध करें",
+    "python.command.python.startREPL.title": "REPL शुरू करें",
+    "python.command.python.createTerminal.title": "टर्मिनल बनाएं",
+    "python.command.python.buildWorkspaceSymbols.title": "वर्कस्पेस सिंबल बनाएं",
+    "python.command.python.runtests.title": "सभी टेस्ट चलाएं",
+    "python.command.python.debugtests.title": "सभी टेस्ट डीबग करें",
+    "python.command.python.execInTerminal.title": "टर्मिनल में पायथन फाइल चलाएं",
+    "python.command.python.setInterpreter.title": "इंटरप्रेटर चुनें",
+    "python.command.python.refactorExtractVariable.title": "Variable निकालें",
+    "python.command.python.refactorExtractMethod.title": "Method निकालें",
+    "python.command.python.viewTestOutput.title": "टेस्ट आउटपुट दिखाएँ",
+    "python.command.python.selectAndRunTestMethod.title": "टेस्ट विधि चलाएँ ...",
+    "python.command.python.selectAndDebugTestMethod.title": "डिबग टेस्ट विधि ...",
+    "python.command.python.selectAndRunTestFile.title": "टेस्ट फ़ाइल चलाएँ ...",
+    "python.command.python.runCurrentTestFile.title": "वर्तमान परीक्षण फ़ाइल चलाएँ",
```

```suggestion
    "python.command.python.runCurrentTestFile.title": "वर्तमान टेस्ट फ़ाइल चलाएँ",
```
Blocking on परीक्षण => टेस्ट
Non blocking, मौजूदा might be the right word here वर्तमान is particularly used for time